docs/source/deployguide/ca-config.md

```diff
@@ -0,0 +1,420 @@
+
+- **`tls.clientauth.certfiles`**: For mTLS only, provide the PEM-encoded list of root certificate authorities that the server uses when verifying client certificates.
+
+## cafiles
+
+```
+cafiles:
+```
+
+As mentioned in the topic on [Planning for a CA](ca-deploy-topology.html), the `cafiles` parameter can be used to configure a dual-headed CA -- a single CA that under the covers includes both a root CA and a TLS CA. This usage pattern can be for convenience, allowing each CA to maintain its own configuration but still share the same back-end user database. In the `cafiles` parameter, enter the path to the `fabric-ca-server-config.yaml` of the second CA server, for example the TLS CA. The configuration of the secondary CA can contain all of the same elements as are found in the primary CA server config file except `port`, `ca.name`, and `tls` sections.
+
+If this is not a desired configuration for your CA, you can leave the value of this parameter blank.
+
+## intermediate CA
+```
+intermediate:
+  parentserver:
+    url:
+    caname:
+
+  enrollment:
+    hosts:
+    profile:
+    label:
+
+  tls:
+    certfiles:
+    client:
+      certfile:
+      keyfile:
+```
+
+Intermediate CAs are not required, but to reduce the risk of your root CA becoming compromised, you may want to include one or more intermediate CAs in your network.
+
+**Important:** Before setting up an intermediate CA, you need to verify the value of the `csr.ca.pathlength` parameter in the parent CA. When set to `0`, the root CA can issue intermediate CA certificates, but these intermediate CAs may not in turn enroll other intermediate CAs. If you want your intermediate CA to be able to enroll other intermediate CAs, the root ca `csr.ca.pathlength` needs to be set to `1`.  And if you want those intermediate CAs to enroll other intermediate CAs, the root ca `csr.ca.pathlength` would need to be set to `2`.
+
+- **`parentserver.url`**: Specify the parent server url in the format `https://<PARENT-CA-ENROLL-ID>:<PARENT-CA-SECRET>@<PARENT-CA-URL>:<PARENT-CA-PORT>`.
+- **`parentserver.caname`**: Enter `ca.name` of the parent CA server.
+- **`enrollment.profile:`**: Enter the value of the `signing.profile` for the parent CA. Normally this would be `ca`.
+- **`tls.certfiles`**: Enter the location and name of the TLA CA signing cert, `ca-cert.pem` file. For example, `tls/ca-cert.pem`. This location is relative to where the server configuration .yaml file exists.
+
+In addition to editing this `intermediate` section, you also need to edit the following sections of the configuration .yaml file for this intermediate CA:
+- `csr` - Ensure that the `csr.cn` field is blank.
+- `port` - Be sure to set a unique port for the intermediate CA.
+- `signing` - Verify that `isca` set to `true`, and `maxpathlen` is set to greater than `0` only if this intermediate CA will serve as a parent CA to other intermediate CAs, otherwise it should be set to `0`.
```

Comment:
I found this out the other day, to enforce a max path length of zero in CFSSL which is a dependency of CA, you need to set `maxpathlenzero` to `true`. The maxpathlen should be used when setting a value of 1 or greater, with `maxpathlenzero` being `false`.

Reply:
Added in signing section.